dependencies_ros_melodic.rosinstall

```diff
@@ -1,3 +1,3 @@
 # PCL ros tools, this is required because using custom version of PCL and VTK
 - git: {local-name: perception_pcl, uri: 'https://github.com/ros-perception/perception_pcl.git', version: 1.7.1}
-- git: {local-name: ros_industrial_cmake_boilerplate, uri: 'https://github.com/ros-industrial/ros_industrial_cmake_boilerplate.git', version: 0.2.8}
+- git: {local-name: ros_industrial_cmake_boilerplate, uri: 'https://github.com/ros-industrial/ros_industrial_cmake_boilerplate.git', version: 0.2.15}
```

Comment:
Is the new version backwards-compatible?  Or could this cause incompatibilities across projects?

Reply:
I believe so. I think we are tracking patch changes on that repository correctly. I've tested for a few projects of my own, and this hasn't broken anything for me yet.